language/sygus.tex

```diff
@@ -2334,8 +2334,11 @@ \section{Calling Oracles}%
 } of sorts $\sigma_1, \ldots, \sigma_n$
 written using the syntax for values described in this document.
 Let ${\tt oracle}$ be the name of the binary corresponding to the oracle.
-When the oracle is executed with the command
-${\tt oracle textfile.oracle}$,
+When an oracle that is queried by command line is executed with the command
+${\tt oracle \,\, \paren{v_1 \ldots v_n}}$,
```

Comment:
I think we should drop the parenthesis when calling on the command line, what do you think?

Also, I think this definition needs to be split properly into two parts "implements an interface via command line" and "implements an interface via files".

Reply:
ok i've split this. It's a bit repetitiveve